dist/main.js

```diff
@@ -0,0 +1,37 @@
+$('#getTeam').on('click', function() {
+    const name = $('#input').val();
+    const url = `/teams/${name}`;
+    apiManager.getData(url, rendererTeam.render, name, rendererTeam.wrongData);
+    $('#input').val('');
+});
+
+$('#dream-team-button').on('click', function() {
+    const url = '/dreamTeam';
+    apiManager.getData(url, rendererDream.render, 'Your Dream Team', rendererTeam.noDream);
```

Comment:
the apiManager shouldn't get the render method, it should just concern in getting the data from the external API and store it in it's `data` property. 
Afterwards, the render should be invoked with the data of the apiManager - just like in the tweeter project, but in this case because the calls are async, the invocation of the render method should either be in a callback or in a `.then` function